sdk/cosmosdb/cosmos/CHANGELOG.md

```diff
@@ -27,6 +27,37 @@ const containerDefinition = {
 database.container.create(containerDefinition);
 ```
 
+- FEATURE: Adds support for autopilot parameters on container and database create methods
+
+Note that maxThroughput and autoUpgradePolicy must be specified together, and cannot be passed with `throughput`
+
+```js
+// correct
+const containerDefintion = {
```

Comment:
I would make these examples `container.create` calls. Usage less clear when this are just objects